### DISCUSSION TITLE
Fitur tambahan

### DISCUSSION THREAD

**Discussion by ImamFattahillah** [ikhsan3adi/absensi-sekolah-qr-code#76] [General]
Apakah tidak bisa ditambah fitur utk report ke no WA ortu bahwa anaknya sudah masuk dan sudah pulang sekolah?

**Comment by reactmore** [comment]
Bisa saja tapi karena ini service wa , sms gk gratis mungkin jadi optional saja, untuk yang free seperti email dan telegram saya ada niat tambahin juga

**Reply by nanda443** [reply]
Kira kira kapan ya kak fiturnya dibuat?

**Reply by reactmore** [reply]
Minggu-minggu ini kayanya sambil nyicil di branch lokal 😂

**Reply by nanda443** [reply]
Ditunggu ya kak😅biar bisa saya implementasi di sekolah🙏😅

**Comment by a-alrizky-a** [comment]
Cocok saya juga ada Wagateway yang non official siapa tau ada rencana kesana minimal API
